.github/workflows/draft.yml

```diff
@@ -39,6 +58,22 @@ jobs:
         if: matrix.target == 'riscv64gc-unknown-linux-gnu'
         run: sudo apt update && sudo apt install -yq gcc-riscv64-linux-gnu
 
+      - name: Install gcc-powerpc64-linux-gnu
+        if: matrix.target == 'powerpc64-unknown-linux-gnu'
+        run: sudo apt update && sudo apt install -yq gcc-powerpc64-linux-gnu
+
+      - name: Install gcc-powerpc64le-linux-gnu
+        if: matrix.target == 'powerpc64le-unknown-linux-gnu'
+        run: sudo apt update && sudo apt install -yq gcc-powerpc64le-linux-gnu
+
+      - name: Install gcc-sparc64-linux-gnu
+        if: matrix.target == 'sparc64-unknown-linux-gnu'
+        run: sudo apt update && sudo apt install -yq gcc-sparc64-linux-gnu
+
+      - name: Install gcc-s390x-linux-gnu
+        if: matrix.target == 's390x-unknown-linux-gnu'
+        run: sudo apt update && sudo apt install -yq gcc-s390x-linux-gnu
+
```

Comment:
Please merge these into one, i.e. check `matrix.target` before running `sudo apt install` in the shell